README.md

```diff
@@ -10,22 +10,22 @@ _How do I, as a developer, start working on the project?_
 This project depends on:
 
 1. Drupal 8
-1. [composer](https://getcomposer.org/download/)
-1. [Node.js](https://nodejs.org/en/download/)
-1. [Homebrew](http://brew.sh/)
-1. pbcopy & pbpaste (yep because I use OS X)
-1. cowsay (```brew install cowsay```)
-1. [phpcs](http://pear.php.net/package/PHP_CodeSniffer/) and [Coder](https://www.drupal.org/project/coder) - see https://www.drupal.org/node/1419988 for instructions.
-1. [eslint](http://eslint.org/docs/user-guide/getting-started)
-1. [phpunit](https://phpunit.de/getting-started.html)
+2. [composer](https://getcomposer.org/download/)
+3. [Node.js](https://nodejs.org/en/download/)
+4. [Homebrew](http://brew.sh/) (only for OS X)
+5. pbcopy & pbpaste (for linux see http://superuser.com/a/288333/143345)
+6. cowsay (OS X `brew install cowsay` or in Ubuntu `apt-get install cowsay`)
+7. [phpcs](http://pear.php.net/package/PHP_CodeSniffer/) and [Coder](https://www.drupal.org/project/coder) - see https://www.drupal.org/node/1419988 for instructions.
+8. [eslint](http://eslint.org/docs/user-guide/getting-started)
+9. [phpunit](https://phpunit.de/getting-started.html)
```

Comment:
The line numbering is actually a feature of Markdown. You can number each entry as `1.` and it will output consecutive numbers. This makes it easy to maintain :)